docs/BreakingChanges/147 BinaryFormatter can fail to find type from LoadFrom context.md

```diff
@@ -0,0 +1,44 @@
+## 147: BinaryFormatter can fail to find type from LoadFrom context
+
+### Scope
+Edge
+
+### Version Introduced
+4.5
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+As of .NET Framework 4.5, a number of XmlSerializer changes may cause differences in deserialization when using `BinaryFormatter` to deserialize types that had been loaded in the
+LoadFrom context. The default serialization binder does not automatically search the LoadFrom context, although it may have worked in some circumstances based on the old behavior
+of XmlSerializer. Due to the changes, a FileNotFoundException may be thrown that the Assembly it was defined in could not be found.
```

Comment:
> Due to the changes, a FileNotFoundException may be thrown that the Assembly it was defined in could not be found.

I am not sure I understand this sentence. Does it mean sometimes a FileNotFoundException is thrown when it cannot find the assembly that the class was defined in?